examples/gno.land/r/sys/params/unlock.gno

```diff
@@ -16,3 +23,44 @@ func ProposeUnlockTransferRequest() dao.ProposalRequest {
 func ProposeLockTransferRequest() dao.ProposalRequest {
 	return NewSysParamStringsPropRequestWithTitle(bankModulePrefix, "p", restrictedDenomsKey, lockTransferTitle, []string{"ugnot"})
 }
+
+func ProposeAddUnrestrictedAcctsRequest(addrs ...std.Address) dao.ProposalRequest {
```

Comment:
Check the current discussion with @moul, we can simplify this 🙏

Reply:
Pease see my comments below.